src/components/todos/todo-list-table.tsx

```
@@ -57,49 +62,73 @@ const TodoListTableRow = ({
+          <TableCell className="whitespace-nowrap">
+            {todo.deferredDetails?.deferredTill
+              ? new DateUtil(todo.deferredDetails.deferredTill).format(DateFormats.D_MMM_YYYY)
+              : '--'}
+          </TableCell>
         )}
-      </TableCell>
-    </TableRow>
+
+        <TableCell>
+          {showActions ? (
+            <div className="flex items-center gap-0.5" onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)